build.gradle

```diff
@@ -198,24 +198,16 @@ subprojects {
     integrationTest {
       java.srcDir file('src/integration-test/java')
       resources.srcDir file('src/integration-test/resources')
+      compileClasspath += sourceSets.main.output + sourceSets.test.output
+      runtimeClasspath += sourceSets.main.output + sourceSets.test.output
     }
   }
 
   configurations {
     integrationTestImplementation.extendsFrom testImplementation
-    integrationTestImplementation.setCanBeResolved(true)
```

Comment:
Ah, I’m curious to what IDE you are using to develop with? Digging back in history a bit (#2961, #2375), I think some of this `integrationTestImplementation` setup may have been added to work around some Eclipse-specific issues, but I don’t personally use Eclipse for development and don’t have a quick way to confirm. It would be great to know if they no longer causing problems though!

Reply:
I use Intellij, which didn't need this. If it's required for Eclipse, keeping it seems reasonable.

Reply:
Let's keep it in if it doesn't hurt.

Reply:
Sounds good